### ISSUE TITLE
validator uses 'any' %version

### ISSUE THREAD

**Issue by bmwiedemann** [openSUSE/obs-service-source_validator#33]
There seems to be a bug in source_validator, which can be triggered with

``` bash
osc co openSUSE:Factory/yudit
cd $_
osc build
```

```
Run source service: /usr/lib/obs/service/format_spec_file --outdir /tmp/tmpXxGi7C
Run source service: /usr/lib/obs/service/source_validator --outdir /tmp/tmpYiyjeJ
(E) yudit-any.tar.bz2 mentioned in spec file does not exist.
(W) Attention, yudit-2.9.2.tar.bz2 is not mentioned in spec files as source or patch.
Aborting: service call failed:  /usr/lib/obs/service/source_validator --outdir /tmp/tmpYiyjeJ
Source service run failed!
```

and can be workarounded by re-ordering spec file entries like this:
https://build.opensuse.org/request/show/365934

**Comment by marcus-h** [comment]
This is fixed by PR#39 (actually, just a side-effect:) ).

**Comment by scarabeusiv** [comment]
PR #39 was merged.
